content/docs/other-guides/upgrade.md

```diff
@@ -12,31 +12,25 @@ deployments.
 
 Updating your deployment is a two step process:
 
-1. Update your ksonnet application:
+1. Update your kustomize manifests:
 
-  1. We recommend checking your app into source control to back it up before 
+  1. We recommend checking your local packages into source control to back it up before 
      proceeding.
-  1. Use the script 
-     [upgrade_ks_app.py](https://github.com/kubeflow/kubeflow/tree/{{< params "githubbranch" >}}/scripts)
-     to update your ksonnet app with the current version for the Kubeflow 
-     packages.
-
-     Note: ksonnet is working on support for this capability. 
-     See https://github.com/ksonnet/ksonnet/issues/237
+  1. Use `kfctl` to download the desired version of Kubeflow, e.g.:
+     ```
```

Comment:
This code block isn't rendering properly. See preview: https://deploy-preview-928--competent-brattain-de2d6d.netlify.com/docs/other-guides/upgrade/

To fix, add a blank line before the code block and indent the code block by 2 more spaces. Then add another blank line before the sentence beginning with "Note".